mklive.sh.in

```diff
@@ -282,7 +308,7 @@ generate_iso_image() {
 #
 # main()
 #
-while getopts "a:b:r:c:C:T:Kk:l:i:I:s:S:o:p:v:h" opt; do
+while getopts "a:b:r:c:C:T:Kk:l:i:I:s:S:o:p:v:d:t:h" opt; do
```

Comment:
```suggestion
while getopts "a:b:r:c:C:T:Kk:l:i:I:s:o:p:v:d:t:h" opt; do
```
Sorry! I have [removed the `-S` option](https://github.com/void-linux/void-mklive/pull/214), so there is a conflict now.